staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -198,7 +198,13 @@ func (p *Patcher) Patch(current runtime.Object, modified []byte, source, namespa
 		}
 		patchBytes, patchObject, err = p.patchSimple(current, modified, source, namespace, name, errOut)
 	}
-	if err != nil && (errors.IsConflict(err) || errors.IsInvalid(err)) && p.Force {
+	var forbiddenTransitions = true
```

Comment:
drop this change for now... we're not ready to make client-side changes yet. since n-1 servers do not return CauseTypeFieldChangeForbidden errors, this would make kubectl skip delete-and-recreate behavior against skewed older servers in a way that could be considered breaking behavior

Reply:
see comment at https://github.com/kubernetes/kubernetes/pull/108578/files#r821921891 about the ordering we'll need to roll this change out in